code/src/main/java/com/onlinecontacttracing/messaging/MessageServlet.java

```diff
@@ -0,0 +1,97 @@
+package com.onlinecontacttracing.messaging;
+
+import com.onlinecontacttracing.messaging.LocalityResource;
+import com.onlinecontacttracing.storage.CustomizableMessage;
+import com.onlinecontacttracing.storage.PositiveUser;
+import com.onlinecontacttracing.storage.PotentialContact;
+import com.onlinecontacttracing.messaging.filters.CheckMessagesForFlags;
+import java.util.List;
+
+
```

Comment:
Whitespace. You should have some space between static imports and regular imports, but otherwise put them together.